docs/getting-started/development-environment.md

```diff
@@ -26,10 +26,12 @@ You can now view example-app in the browser.
 Run the following commands if you have Docker environment
 
 ```bash
-$ docker build . -t spotify/backstage
+$ yarn docker-build
 $ docker run --rm -it -p 80:80 spotify/backstage
 ```
 
+> See [package.json](package.json) for other yarn commands/options.
```

Comment:
This doesn't link to the root pkg json. I think you can use `/package.json` instead, or make it relative. Probably move it below the next line though, since it relates to the docker command we just ran.